### ISSUE TITLE
Refine Conforms To display

### ISSUE THREAD

**Issue by gautierchomel** [w3c/publ-a11y#138]
The _Conforms To_ information is actually a technical information like _EPUB Accessibility 1.0 - WCAG 2.0 Level AA_ and maybe with time the users will take habit of it, but it looks important to have a more human readable information.

Proposal is to add a label to each key as follow: 
* No accessibility conformance declared 
* Basic accessibility (EPUB accessibility A)
* Sufficient accessibility (EPUB accessibility AA)
* Advanced accessibility (EPUB accessibility AAA)

Related to #136 and #137

**Comment by clapierre** [comment]
We have discussed this in the past and decided against this approach.  First off one can never be truly AAA accessible.
Besides one won't understand the subtle differences between Sufficiently accessible vs. Advance Accessible.
I think we just declare what this publication conforms to and leave it at that.

**Comment by gregoriopellegrino** [comment]
One approach we have thought about in LIA is to decline this information according to the audience to whom it will be shown and the local regulations.

For example: in Europe it is relevant to know whether a title is in line with the requirements of the European Accessibility Act or not, in Canada there are other requirements, in Brazil there might be additional ones, etc.

Our proposal is to use the _conformsTo_ metadata (and eventually other metadata) to define whether or not a title is in line with local requirements and show this information to the end user. Of course, a solution would have to be provided for all states where there are no local regulations.

**Comment by HadrienGardeur** [comment]
@gregoriopellegrino how would this work? We would define additional values for `conformsTo`?

I personally find the recommended approach in EPUB Accessibility 1.1 confusing and potentially problematic. Moving away from URI to a pseudo text-template kills the built-in extensibility of allowing any URI in `conformsTo`. There's also a higher risk of parsing such text templates incorrectly, which wouldn't match known values.

**Comment by gregoriopellegrino** [comment]
> @gregoriopellegrino how would this work? We would define additional values for `conformsTo`?

Our idea would be to map the values we already have (without additions) to local regulations. For example, for the European Accessibility Act:

- if `conformsTo` is equal to `EPUB Accessibility 1.1 - WCAG 2.1 Level AA`, then it is in line with the European Accessibility Act
- if `conformsTo` is equal to `EPUB Accessibility 1.1 - WCAG 2.1 Level A` plus the content redering settings can be changed by the user (`displayTransformability`) and there are no color contrast issues, then it is in line with the European Accessibility Act
- etc.

**Comment by LaZay** [comment]
> 

I full agree. I have never seen such a regression in a digital standardisation in my whole carrer.

**Comment by LaZay** [comment]
Even the technical values we have so far are not appropriate, for different reasons.

1. The former URIs used for static combinations has been newly replaced by strings, which is a big regression: using strings for IDs is a bad practice in computer science.
2. ARIA is missing.
3. Versions of the involved standards do regularly change, and not necessarily all together, at the same date. Static combination (instead of dynamic one) make it even more difficult to adopt the last version of EACH separate accessibility standard.
4. No accessibility expert nor impaired person knows about the EPUB a11y spec, as it is still **not a W3C REC** yet. As matter of fact, all we have from the W3C so far is this:
- https://www.w3.org/Submission/2017/SUBM-epub-a11y-20170125 (member submission)
- https://www.w3.org/Submission/2017/SUBM-epub-a11y-20170125 (member submission)
- https://www.w3.org/TR/2022/CRD-epub-a11y-11-20221124 (candidate recommendation)
- https://www.w3.org/TR/2022/NOTE-epub-a11y-tech-11-20220913 (group note)

I suggest we first improve our **technical conformance labels**, in order to separate technical conformance to each a11y standard (and help for dynamic combination for model pupils):
1. Conformance to an EPUB A11Y spec version
2. Conformance to a WCAG spec version and level
3. Conformance to an ARIA version

Examples of normalized technical conformance values for machines (not humans) (=> URIs=URLs):

- https://www.w3.org/TR/2022/CRD-epub-a11y-11-20221124 (REC to come)
- https://www.w3.org/TR/2022/CR-WCAG22-20220906#cc1-levelA (A level conformance anchor to be added in forthcoming 2.2 REC)
- https://www.w3.org/TR/2022/CR-WCAG22-20220906#cc1-levelAA (AA level conformance anchor to be added in forthcoming 2.2 REC)
- https://www.w3.org/TR/2022/CR-WCAG22-20220906#cc1-levelAAA (AAA level conformance anchor to be added in forthcoming 2.2 REC)
- https://www.w3.org/TR/2017/REC-wai-aria-1.1-20171214

Knowing that, in the EPUB-OPF metadata model, the _ConformsTo_ property is repeatable.

Furthermore, French publishers, and French Ministry of Culture are wondering if a **conformance to a legal texts that contain exemptions** (European Directive + national transpositions) is of any interest for impaired people. What kind of conformance will we address in this case? **Legal conformance**? Or **functional conformance**? Legal conformance would be misleading, as not accessible products could be conformant if the exception were admissible. Only functional conformance, **free of ANY exemption**, would be usefull, and acceptable by impaired users. How to say that in simple words to a large audience on digital boostores/libraries? What URI (if we do restore them) / normalized string (if not) should we put in our OPF metadata? 

If you refer to the [EPUB Accessibility - EU Accessibility Act Mapping ](https://www.w3.org/TR/epub-a11y-eaa-mapping), I am afraid that this administrative document were dedicated to the commission, in order to prove that the EPUB format was an accessible digital book format (eligible to the act). Nobody knows about this Accessibility Act Mapping. Plus, as it is a mapping document, it is not self-sufficient: by construction, it constantly refers to the European Act itself. It can help us calculate the conformance to the European Act. But that's it.

**Comment by LaZay** [comment]
> 
Why this restriction on AAA? I do not see any technical reason for this on my side. The reason is mainly the cost and the competency.
What if Education publishers decide for a new innovative offer, to answer the need for a fully distant eLearning? They would not be able to signal their little jewels?!

**Comment by LaZay** [comment]
Why this restriction on AAA? I do not see any technical reason for this on my side. The reason is mainly the cost and the competency.
Furthermore, if Education publishers decided for a new innovative high level offer (to answer the need for a fully distant teaching, they would not be able to signal their little jewels?! Why prevent this in an accessibility standard?

**Comment by gregoriopellegrino** [comment]
> Why this restriction on AAA? I do not see any technical reason for this on my side. The reason is mainly the cost and the competency. Furthermore, if Education publishers decided for a new innovative high level offer (to answer the need for a fully distant teaching), they would not be able to signal their little jewels?! Why prevent this in an accessibility standard?

Dear Laurence I suggest you to open a new issue on this topic, this way we can manage it in a simpler way.

**Comment by LaZay** [comment]
Done.  AAA issue move to [https://github.com/w3c/publ-a11y/issues/141](https://github.com/w3c/publ-a11y/issues/141)

**Comment by gautierchomel** [comment]
Conforms To has been refined (see  [3.7.2 Conformance in 15th november draft](https://rawcdn.githack.com/w3c/publ-a11y/f7f9921cde19fd819683e9ea420482577f84088f/UX-Guide-Metadata/draft/principles/index.html#conformance-group)).
